src/Console/Command/NewVersionChecker.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Console\Command;
+
+/**
+ * @internal
+ */
+class NewVersionChecker
```

Comment:
can be `final` I think? and feel free to `@author` yourself ;)

Reply:
Class made `final`.